js/common.js

```diff
@@ -765,40 +765,40 @@ Timer.prototype.stop = function()
 	this.interval = (new Date()).getTime() - this.initial;
 };
 
-var theTabs =
-{
-	tabs:
-	{
-   		gcont : theUILang.General,
-   		FileList : theUILang.Files,
-   		TrackerList : theUILang.Trackers,
-   		PeerList : theUILang.Peers,
-   		Speed : theUILang.Speed,
-   		PluginList : theUILang.Plugins,
-   		lcont : theUILang.Logger
-   	},
+var theTabs = {
+	tabs: {
+		gcont: theUILang.General,
+		FileList: theUILang.Files,
+		TrackerList: theUILang.Trackers,
+		PeerList: theUILang.Peers,
+		Speed: theUILang.Speed,
+		PluginList: theUILang.Plugins,
+		lcont: theUILang.Logger
+  },
 
-   	init: function()
-   	{
-		if(browser.isKonqueror && (browser.versionMajor<4))
-		{
+	init: function() {
+		if (browser.isKonqueror && (browser.versionMajor<4)) {
 			delete this.tabs["Speed"];
 			$("#Speed").remove();
 		}
```

Comment:
I would like to drop this code block. Konqueror is up to version 24 major now. This is over 10 years outdated.